lib/java/src/org/apache/thrift/server/TThreadPoolServer.java

```diff
@@ -29,13 +29,13 @@
 import org.apache.thrift.TException;
 import org.apache.thrift.TProcessor;
 import org.apache.thrift.protocol.TProtocol;
-import org.apache.thrift.transport.TSaslTransportException;
 import org.apache.thrift.transport.TServerTransport;
 import org.apache.thrift.transport.TTransport;
 import org.apache.thrift.transport.TTransportException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.thrift.transport.TTransportException.END_OF_FILE;
```

Comment:
TTransportException is already imported - suggest using TTransportException.END_OF_FILE in your comparisons to keep with the coding style used in the file.

Reply:
Right you are. I've changed this to use `TTransportException.END_OF_FILE` instead.

Reply:
In the course of resolving \#2 above, I noticed the following discrepancy, between the C++ and Java `TTransportException` types:

In C++:
```cpp
// TTransportException.h
  enum TTransportExceptionType {
    UNKNOWN = 0,
    NOT_OPEN = 1,
    TIMED_OUT = 2,
    END_OF_FILE = 3,
    INTERRUPTED = 4,
    BAD_ARGS = 5,
    CORRUPTED_DATA = 6,
    INTERNAL_ERROR = 7
  };
```
From the corresponding Java class:
```java
// TTransportException.java
  public static final int UNKNOWN = 0;
  public static final int NOT_OPEN = 1;
  public static final int ALREADY_OPEN = 2;
  public static final int TIMED_OUT = 3;
  public static final int END_OF_FILE = 4;
  public static final int CORRUPTED_DATA = 5;
```

`TIMED_OUT` and `END_OF_FILE` are off by 1, and `INTERRUPTED` isn't available among the Java types. This might need fixing separately. For the moment, I'll be muting `TIMED_OUT` and `END_OF_FILE`.